src/components/TextMaskedInput/TextMaskedInput.js

```diff
@@ -135,7 +135,9 @@ export const TextMaskedInput = (props) => {
 
   return (
     <>
-      <InputLabel name={name} labelCopy={labelCopy} allCaps={allCaps} />
+      {labelCopy && (
```

Comment:
Should we go through all the fields and do this then? @stephaniel96 is about to jump through several hoops (if she didn't already) to get around this for Nora as well for the TextAreaInput. We also needed to expose some other stuff too specific for Nora so wrapping it in a `NoraTextAreaInput` might make sense anyway, but, I think we'll need to make labelCopy optional for all fields. Can you do an audit and advise on a game plan for follow ups and/or tickets?

Reply:
I'm actually doing the same in one of my PRs -- I'm passing in `labelCopy=''` into a few components (specifically `BirthdateInput`,`ZipInput`, `TextInput`, but I'm still adding more)

Reply:
My current WIP is #272 and would be more than happy to make the label optional for the `TextAreaInput` component